cluster-up/cluster/kind-1.23/provider.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e
+
+DEFAULT_CLUSTER_NAME="kind-1.23"
+DEFAULT_HOST_PORT=5000
+ALTERNATE_HOST_PORT=5001
+export CLUSTER_NAME=${CLUSTER_NAME:-$DEFAULT_CLUSTER_NAME}
+
+if [ $CLUSTER_NAME == $DEFAULT_CLUSTER_NAME ]; then
+    export HOST_PORT=$DEFAULT_HOST_PORT
+else
+    export HOST_PORT=$ALTERNATE_HOST_PORT
+fi
+
+function set_kind_params() {
+    export KIND_VERSION="${KIND_VERSION:-0.12.0}"
+    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-quay.io/kubevirtci/kindest_node:v1.23.3@sha256:0df8215895129c0d3221cda19847d1296c4f29ec93487339149333bd9d899e5a}"
```

Comment:
Aint this version contradicts what you wrote on the PR desc?
btw if you want a multi arch image, we can mirror it to quay, now when we mirror it supports multi arch

Reply:
> Aint this version contradicts what you wrote on the PR desc? 

Do you mean that kind-1.23 contradicts with KIND_VERSION which is 0.12.0? 1.23 refer to kubernetes version rather than kind version, and here it is related to the KIND_NODE_IMAGE tag which is v1.23.3.

> btw if you want a multi arch image, we can mirror it to quay, now when we mirror it supports multi arch you need to add it here if you want https://github.com/kubevirt/project-infra/blob/main/hack/images_to_mirror.csv

Is it means that I only need to put the image name (like following line) in the csv document, then multi-arch image would automatically be pull?
```
docker.io,kindest/node:v1.23.4,quay.io/kubevirtci
```

Reply:
> > Aint this version contradicts what you wrote on the PR desc?
> 
> Do you mean that kind-1.23 contradicts with KIND_VERSION which is 0.12.0? 1.23 refer to kubernetes version rather than kind version, and here it is related to the KIND_NODE_IMAGE tag which is v1.23.3.

"(The default KIND_NODE_IMAGE, quay.io/kubevirtci/kindest_node:v1.23.3, does not support Arm64)"
I mean that this line of the PR desc, is the actual line that you use, don't you want an image which is multi arch ?

> 
> > btw if you want a multi arch image, we can mirror it to quay, now when we mirror it supports multi arch you need to add it here if you want https://github.com/kubevirt/project-infra/blob/main/hack/images_to_mirror.csv
> 
> Is it means that I only need to put the image name (like following line) in the csv document, then multi-arch image would automatically be pull?
> 
> ```
> docker.io,kindest/node:v1.23.4,quay.io/kubevirtci
> ```

A post submit job would push it, at the first time a repo is created under quay, one of the maintainer might need to make it public, but once those two are done you will be able to use it, yes, and it will be multi arch

Reply:
I see, thanks. I will submit a PR for it.